packages/commercetools/theme/components/AppHeader.vue

```diff
@@ -30,17 +31,24 @@
 
 <script>
 import { SfHeader, SfImage } from '@storefront-ui/vue'
-import { toggleCartSidebar } from '../assets/ui-state'
+import { toggleCartSidebar, toggleLoginModalOpen, isLoggedIn } from '../assets/ui-state'
```

Comment:
i just noticed that I left it in thsi form, we should use aliases instead for readability
```suggestion
import { toggleCartSidebar, toggleLoginModalOpen, isLoggedIn } from '~/assets/ui-state'
```